code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -272,28 +272,44 @@ var/global/chicken_count = 0
 	else
 		SSobj.processing.Remove(src)
 
-/obj/udder
 
-/obj/udder/New()
+/obj/item/udder

```

Comment:
And this is exactly the reason why it "constantly adds it's reagent to your blood".